Sources/StreamChatUI/Appearance+Images.swift

```diff
@@ -270,6 +270,46 @@ public extension Appearance {
             }
         }()
 
+        public var menuKeyboard: UIImage? = {
+            if #available(iOS 13.0, *) {
+                return UIImage(systemName: "keyboard")?.withRenderingMode(.alwaysTemplate)
+            } else {
+                return nil
+            }
+        }()
+
+        public var addIcon: UIImage? = {
+            if #available(iOS 13.0, *) {
+                return UIImage(systemName: "plus")?.withRenderingMode(.alwaysTemplate)
+            } else {
+                return nil
+            }
+        }()
+
+        public var downloadSticker: UIImage? = {
+            if #available(iOS 13.0, *) {
+                return UIImage(systemName: "arrow.down.circle")?.withRenderingMode(.alwaysTemplate)
+            } else {
+                return nil
+            }
+        }()
+
+        public var downloadStickerFill: UIImage? = {
+            if #available(iOS 13.0, *) {
+                return UIImage(systemName: "arrow.down.circle.fill")?.withRenderingMode(.alwaysTemplate)
```

Comment:
@parth-simformsolutions  Please remove rendering mode property.
We can change image tint color without adding any property.